### PR TITLE
[Lora] Fix generate

### LINKED ISSUE #280: lora  TypeError
python lora.py --model mistralai/Mistral-7B-v0.1 \
               --adapter-file adapters.npz \
               --prompt "table: 1-10015132-16 
columns: Player, No., Nationality, Position, Years in Toronto, School/Club Team
Q: What is terrence ross' nationality
A: "

    logits, cache = model(y[None], cache=cache)
                          ~^^^^^^
TypeError: list indices must be integers or slices, not NoneType

### PR DESCRIPTION
**Pull request by awni**
Closes #280 and #281

I'm piggybacking some updates to the whisper example on this as well:
- fix test
- better default model
- Updates to README

### CODE REVIEW

**Comment by awni** [comment]
Closes #277

**Comment by vtboyarc** [comment]
I pulled down this pr and can confirm that the lora TypeError is resolved! Great work, thank you! And for responding to me so fast on Twitter (𝕏) and getting this fix in!

File: whisper/README.md [hunk 1]
```diff
@@ -52,6 +52,16 @@ import whisper
 text = whisper.transcribe(speech_file)["text"]
 ```
 
+Choose the model by setting `hf_path_or_repo`. For example:
+
+```python
+result = whisper.transcribe(speech_file, hf_path_or_repo="models/large")
```
Comment (Vaibhavs10) [inline whisper/README.md:58]: Oh this is lovely! Thanks for adding this! <3

**Review by angeloskath — APPROVED** [review]
👍

**Pull request merged**